### ISSUE TITLE
[IMPORTANT] Getting OTP error 

### ISSUE THREAD

**Issue by Lihaas** [nic-delhi/AarogyaSetu_Android#253]
{1st Screen Shot}When we click back from the OTP screen to enter phone no. screen, Its showing error to going on OTP screen again.

![Screenshot_2020-05-28-09-59-03-487_nic goi aarogyasetu](https://user-images.githubusercontent.com/25553339/83099299-93b25f80-a0ca-11ea-9f04-b264baecd563.png)

**Comment by aravindvnair99** [comment]
@Lihaas I too experienced this bug when I first signed up. But I don't see this bug anymore. Are you on the latest version of the app?

Another thing I noticed is that autofill isn't correctly configured. It fills with "+91" when the autofill also fills the ISD code.

**Comment by Lihaas** [comment]
yes, I am with the latest version, It's not an error it's a bug so it creates a problem sometimes. 

> @Lihaas I too experienced this bug when I first signed up. But I don't see this bug anymore. Are you on the latest version of the app?
> 
> Another thing I noticed is that autofill isn't correctly configured. It fills with "+91" when the autofill also fills the ISD code.

Yes, I am on the latest version, It's not resolve. This bug occurring 8 out of 3 times
